crates/subspace-service/Cargo.toml

```diff
@@ -21,6 +21,7 @@ derive_more = "0.99.17"
 frame-support = { version = "4.0.0-dev", git = "https://github.com/subspace/substrate", rev = "5f0aa1feb7250ac7b8c1b9928f87b2420b530e22" }
 futures = "0.3.21"
 jsonrpsee = { version = "0.13.1", features = ["server"] }
+pallet-executor = { version = "0.1.0", path = "../pallet-executor" }
```

Comment:
I can understand service is generic about Runtime and ExecutorDispatch, why pallet can't be included for a specific project instead of a general framework? 

And if so, what's your suggestion on this https://github.com/subspace/subspace/blob/961104c4b5a846799adf17bd5d8af949d912d19c/crates/subspace-service/src/pool.rs#L131-L133

move `InvalidTransactionCode` to `sp-executor`?

Reply:
Pallets are runtime logic. Even if there is an enum that looks similar to what you need in service, it is not a good reason to include pallet in the service directly.

Reply:
Well, polkadot does use a few pallets in their service. I don't mind either way, if you prefer not to use pallet in the service, then we should move it into `sp-executor`, or you have any other suggestion?

Reply:
Let's move it to `sp-executor` for now